src/jit/codegenarm64.cpp

```diff
@@ -3628,6 +3628,12 @@ void CodeGen::genCodeForCpObj(GenTreeObj* cpObjNode)
                 default:
                     // We have a GC pointer, call the memory barrier.
```

Comment:
Update the comment to:

` // In the case of a GC-Pointer we'll call the ByRef write barrier helper`